hedera-node/hedera-app/src/main/java/com/hedera/node/app/info/MonoNodeInfo.java

```diff
@@ -40,6 +40,6 @@ public MonoNodeInfo(@NonNull com.hedera.node.app.service.mono.context.NodeInfo d
 
     @Override
     public boolean isSelfZeroStake() {
-        return delegate.isSelfZeroStake();
+        return delegate.isSelfZeroWeight();
```

Comment:
I would think change `public boolean isSelfZeroStake()` also to `public boolean isSelfZeroWeight()`. Or just leave `delegate.isSelfZeroStake()`

Reply:
Fixed